docs/content/blog/authelia-traefik-setup-guide/index.md

```diff
@@ -0,0 +1,391 @@
+ ┃  ┣ 📁 config
+ ┃  ┃ ┣ 📄 configuration.yml
+ ┃  ┃ ┗ 📄 users.yml
+ ┃  ┗ 📁 secrets
+ ┣ 📄 compose.yml
+ ┗ 📁 traefik
+    ┣ 📁 config
+    ┃ ┣ 📄 dynamic.yml
+    ┃ ┗ 📄 traefik.yml
+    ┣ 📁 data
+    ┃ ┗ 📄 acme.json
+    ┣ 📁 logs
+    ┗ 📁 secrets
+```
+
+## Traefik and Whoami
+{{< callout context="note" title="Note" icon="outline/info-circle" >}}
+We'll focus on the minimal configuration needed to work with Authelia. For advanced Traefik features and configurations, consult their documentation.
+{{< /callout >}}
+
+Next, we'll set up Traefik as our reverse proxy. For detailed Traefik documentation, refer to the [official Traefik docs](https://doc.traefik.io/traefik/).
+
+#### Docker Compose
+
+
+```yaml{title="compose.yml"}
+services:
+  traefik:
+    image: traefik:latest
+    container_name: traefik
+    restart: unless-stopped
+    security_opt:
+      - no-new-privileges=true
+    networks:
+      proxy: {}
+      authelia:
+        aliases:
+          - '{{< sitevar name="subdomain-authelia" nojs="auth" >}}.{{< sitevar name="domain" nojs="example.com" >}}'
+    ports:
+      - '80:80'
+      - '443:443'
+    environment:
+      TZ: 'America/Los_Angeles' # see below
+      CF_EMAIL_FILE: '/run/secrets/cloudflare_email'
+      CF_DNS_API_TOKEN_FILE: '/run/secrets/cloudflare_api_key'
+    volumes:
+      - '/var/run/docker.sock:/var/run/docker.sock:ro'
+      - './traefik/config/traefik.yml:/traefik.yml:ro'
+      - './traefik/config/dynamic.yml:/dynamic.yml:ro'
+      - './traefik/data/:/data'
+      - './traefik/logs:/logs'
+    secrets:
+      - cloudflare_email
+      - cloudflare_api_key
+    labels:
+      traefik.enable: 'true'
+      traefik.http.routers.dashboard.rule: 'Host(`traefik.{{< sitevar name="domain" nojs="example.com" >}})'
+      traefik.http.routers.dashboard.entrypoints: 'https'
+      traefik.http.routers.dashboard.middlewares: 'authelia@docker'
+      traefik.http.routers.dashboard.service: 'api@internal'
+
+  whoami:
+    image: traefik/whoami
+    restart: unless-stopped
+    container_name: whoami
+    labels:
+      traefik.enable: 'true'
+      traefik.http.routers.whoami.rule: 'Host(`whoami.{{< sitevar name="domain" nojs="example.com" >}}`)'
+      traefik.http.routers.whoami.entrypoints: 'https'
+    networks:
+      proxy: {}
+
+  ## Other Services Go Here
+
+secrets:
+  cloudflare_email:
+    file: ./traefik/secrets/cloudflare_email.txt
+  cloudflare_api_key:
+    file: ./traefik/secrets/cloudflare_api_key.txt
+
+networks:
+  proxy:
+    external: true
+  authelia: {}
+```
+Notes:
+Timezone strings can be found [here](https://go.dev/src/time/zoneinfo_abbrs_windows.go)
+
+#### Basic Traefik Configuration
+Now we configure Traefik.
+The following files contain the minimal Traefik configuration needed for Authelia integration:
+
+```yaml{title="traefik/config/traefik.yml"}
+# Base Traefik configuration
+api:
+  dashboard: true
+  debug: false
+  insecure: false
+
+log:
+  level: INFO
+  filePath: /logs/traefik.log
```

Comment:
Since a log file is defined here, traefik won't log to stdout anymore.
Maybe a hint on how to view these logs would help?

Reply:
I'm somewhat inclined to think it may be more beneficial to log the standard logs to the console and keep the access logs to a log file.